ci/notebooks-digest-updater.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+USER_HASH=$1
+REPO_OWNER=$2
+REPO_NAME="notebooks"
+BRANCH=main
```

Comment:
What is our strategy for the update on a stable/eus branch? We're just update this variable value?

Reply:
Right, in case that we had to update old releases would be a problem. I added logic to use specific branch if needed (as optional)